src/main/webapp/app/exercise/services/exercise.service.ts

```
@@ -590,9 +591,10 @@ export class ExerciseService {
                 break;
             case ExerciseType.TEXT:
             case ExerciseType.FILE_UPLOAD:
-                const textOrFileUploadExercise = exercise as TextExercise & FileUploadExercise;
-                if (textOrFileUploadExercise.exampleSolution) {
-                    exampleSolution = artemisMarkdown.safeHtmlForMarkdown(textOrFileUploadExercise.exampleSolution);
+            case ExerciseType.MATH:
+                const textOrFileUploadOrMathExercise = exercise as TextExercise & FileUploadExercise & MathExercise;
```

Biome (2.4.16)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)